common/model/events.js

```diff
@@ -67,22 +67,20 @@ export type Place = {|
   capacity: ?number
 |}
 
-type IdentifierScheme = 'eventbrite-id';
-
-type Identifier = {|
-  identifierScheme: IdentifierScheme,
-  value: string
-|}
-
 export type Audience = {|
   id: string,
   title: string,
   description: ?string
 |}
 
+/* eslint-disable no-use-before-define */
+export type UiEvent = {|
+  ...Event,
+  backgroundTexture?: string
+|}
```

Comment:
Do we think that this should just be `Event`?
Might be a bit of a refactor, but I thought the logic evolved into:

Event - the model from Prismic
UiEvent - an event that would take some arguments at render time.

An example of this would be Image and UiImage - where you need to specific the `sizesqueries` of the image when you instantiate it on a template level.

Reply:
Ping @davidpmccormick for discussion here too,

Reply:
Maybe I got it wrong, but my thinking was that `UiThing`s were `Thing`s with extra window dressing – the `backgroundTexture` isn't really an `Event` detail, so doesn't (in this view) belong on the `Event` model?

Reply:
Yea, but to create UI vs data modelling seemed a little vague, and meant making decisions without solid guidelines, which takes time.

The way ☝️ makes it easy to fathom.